view/frontend/web/js/customerprice.js

```diff
@@ -86,6 +86,7 @@ define([
                 priceBox = $('[data-price-box=product-id-' + productId + ']')
             }
             priceBox.priceBox({"priceConfig": config});
+            if (tierPrice) priceBox.priceBox('updateProductTierPrice');
```

Comment:
This should be written as follows:
```js
            if (tierPrice) {
                priceBox.priceBox('updateProductTierPrice');
            }

```

The syntax now used is from 10/20 years ago, this is no longer considered a good idea anymore.